relayer/send.py

```
@@ -1,138 +1,124 @@
+        ],
+        "name": "approve",
+        "outputs": [{"name": "", "type": "bool"}],
+        "payable": False,
+        "stateMutability": "nonpayable",
+        "type": "function"
+    },
+]
+
+# Minimal ABI for the UntronTransfers contract (only intron(...) needed here).
+# Replace with the actual snippet of the ABI covering intron(...) or the full ABI as needed.
+UNTRON_ABI = json.load(open("../out/UntronTransfers.json"))["abi"]
```

Ruff (0.8.2)

Use a context manager for opening files

(SIM115)